tests/string-gen.util.ts

```diff
@@ -0,0 +1,24 @@
+import * as fc from 'fast-check';
+
+function StringArbitrary(charArb: fc.Arbitrary<string>, aLength?: number, bLength?: number) {
+  const arrayArb =
+    aLength != null
+      ? bLength != null
+        ? fc.array(charArb, aLength, bLength)
+        : fc.array(charArb, aLength)
+      : fc.array(charArb);
+  return arrayArb.map(tab => tab.join(''));
+}
+
+function spacedArbitrary(charArb: fc.Arbitrary<string>, minLength?: number, maxLength?: number) {
+  const spaceChars = fc.frequency({ arbitrary: charArb, weight: 100 }, { arbitrary: fc.constant(' '), weight: 1 });
```

Comment:
Hmm I would expect more of a 15:1 ratio rather than 100:1?

Reply:
Fixed